core/src/main/scala/io/finch/Error.scala.orig

```diff
@@ -0,0 +1,121 @@
+package io.finch
```

Comment:
Just rebase your branch against the most recent master and you should be fine.

See [here](http://stackoverflow.com/questions/7929369/how-to-rebase-local-branch-with-remote-master) on how to do that.